indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -411,7 +411,9 @@ def _process(self):
         else:
             pdf_class = RegistrantsListToBadgesPDF
         registration_ids = config_params.pop('registration_ids')
-        signals.event.designer.print_badge_template.send(self.template, regform=self.regform)
+        registrations = Registration.query.filter(Registration.id.in_(registration_ids))
```

Comment:
Better add `.all()` here - passing around query objects is a bit odd (also not what the signal's doc says)

Reply:
I agree, `.all()` added